common/containers/Tabs/Swap/components/CurrentRates.tsx

```diff
@@ -7,19 +7,27 @@ import translate from 'translations';
 import { toFixedIfLarger } from 'utils/formatters';
 import './CurrentRates.scss';
 
+interface Props {
+  [id: string]: NormalizedBityRate;
+}
+
 interface State {
   ETHBTCAmount: number;
   ETHREPAmount: number;
   BTCETHAmount: number;
   BTCREPAmount: number;
 }
-export default class CurrentRates extends Component<Pairs, State> {
-  public state = {
-    ETHBTCAmount: 1,
-    ETHREPAmount: 1,
-    BTCETHAmount: 1,
-    BTCREPAmount: 1
-  };
+
+export default class CurrentRates extends Component<Props, State> {
+  constructor(props) {
```

Comment:
most of the codebase follows the `public state = {...` pattern instead of using a constructor